src/components/Book.js

```diff
@@ -1,19 +1,45 @@
+// Book.js
+
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { addBook, removeBook } from '../redux/Books/booksSlice';
+import { checkStatus } from '../redux/categories/categoriesSlice';
+
+function Book({
+  id, title, author,
+}) {
+  const dispatch = useDispatch();
+
+  const handleAddBook = () => {
+    dispatch(addBook({
+      id, title, author,
+    }));
+  };
+
+  const handleRemoveBook = () => {
+    dispatch(removeBook(id));
+  };
+
+  const handleCheckStatus = () => {
+    dispatch(checkStatus(id));
+  };
```

Comment:
- [ ] Great job for adding the method but I think you put it in the wrong file so kindly add the handleAddBook in your form.js component so when the user click on add book the book will be added since when I try to add nothing is added there so kindly fix the issue 🤩✨💯🔥